nat-lab/tests/telio.py

```diff
@@ -1156,6 +1160,37 @@ async def collect_core_dumps(self):
                 os.system(cmd)
 
 
+async def find_files(connection, where, name_pattern):
+    """Wrapper for 'find' command over the connection"""
+
+    try:
+        process = await connection.create_process(
+            ["find", where, "-maxdepth", "1", "-name", name_pattern]
+        ).execute()
+        return process.get_stdout().strip().split()
+    except ProcessExecError:
+        # Expected when 'where' doesn't exist
```

Comment:
If you call this function on directory which doesn't exist it will return empty list instead of raising an exception. This happens for most of our tests and the moose directory (most of our tests have moose disabled).